src/Microsoft.AspNetCore.Http.Connections.Client/Internal/LongPollingTransport.cs

```diff
@@ -95,12 +92,12 @@ public Task StartAsync(Uri url, TransferFormat transferFormat)
                 // This will also cause the poll to return.
                 await SendDeleteRequest(url);
 
-                // This timeout is only to ensure the poll is cleaned up despite a misbehaving server.
-                // It doesn't need to be configurable.
-                _transportCts.CancelAfter(ShutdownTimeout);
+                _transportCts.Cancel();
```

Comment:
@anurse I'm basically reverting that change to wait for graceful shutdown. For 2 reasons:

- The only thing that matters is that we send the request to make sure the server does aggressively closes the connection.
- We need to wait for the other side of the loop to end because when this returns the HttpConnection disposes the http client.
- The tests need to be updated because none of them handle the delete properly meaning things end up hanging for 5 seconds extending the test time.

Reply:
I'm OK with that. My only concern was to avoid logging "failures" when the poll is cancelled due to graceful shutdown. I think we don't log that as an error though so it's fine.